board/safety/safety_hyundai_canfd.h

```diff
@@ -320,7 +334,9 @@ static int hyundai_canfd_fwd_hook(int bus_num, int addr) {
   }
   if (bus_num == 2) {
     // LKAS for HDA2, LFA for HDA1
-    int is_lkas_msg = (((addr == 0x50) || (addr == 0x2a4)) && hyundai_canfd_hda2);
+    int hda2_steer_msg = hyundai_canfd_hda2_alt_steering ? 0x110 : 0x50;
+    int hda2_block_lfa = hyundai_canfd_hda2_alt_steering ? 0x362 : 0x2a4;
+    int is_lkas_msg = ((addr == hda2_steer_msg) || (addr == hda2_block_lfa)) && hyundai_canfd_hda2;
```

Comment:
same here. maybe do it in a new PR, changing these to bools as well. int is confusing

Reply:
I meant to calculate the address in the new PR